win32ss/gdi/ntgdi/freetype.c

```diff
@@ -7062,6 +7063,22 @@ IntExtTextOutW(
         bitSize.cx = realglyph->bitmap.width;
         bitSize.cy = realglyph->bitmap.rows;
 
+        /* Do characters other than space return a bitSize.cx of zero? */
+        if (ch0 != 32 && bitSize.cx == 0)
+            DPRINT1("WARNING: WChar 0x%04x has a bitSize.cx of zero\n", ch0);
+
+        /* Don't ignore spaces when computing offset.
+         * This completes the fix of CORE-11787.*/
```

Comment:
```suggestion
         * This completes the fix of CORE-11787. */
```
Style nit.

Reply:
Note:
"Requirement" of ending full stop is somewhat "unclear". They may want to make this point explicit.
https://reactos.org/wiki/Coding_Style#Commenting

Reply:
Technically here the comment contains more than one sentence, and the previous sentence already ended with a full-stop. So it would be more than logical to have it.

Reply:
Changed. Thanks.